test/README.md

```diff
@@ -0,0 +1,58 @@
+# Cluster Validation Testing in Bedrock
+
+## Summary
+
+This section describes how to build integration and validation tests for your bedrock cluster environments using the Go testing packages and the terratest modules.
+
+Terratest is a Go library that makes it easier to write automated tests for your infrastructure code. It provides a variety of helper functions and patterns for common infrastructure testing tasks.
+
+In addition, the bedrock test suite allows for better collaboration with embedding into CI/CD tools such as Travis or Azure DevOps Pipelines.
+
+## Prerequisites
+
+Please [install bedrock required tools](/cluster/README.md/#required-tools) in addition to the following:
+
+- [Go](https://golang.org/doc/install)
+- [Dep](https://github.com/golang/go/wiki/PackageManagementTools)
+- [Terratest Modules](https://github.com/gruntwork-io/terratest)
+- CI/CD Tool (Optional)
+
+## Test Setup Locally
+
+In this example we are using the [`azure-simple`](/cluster/environments/azure-simple/readme.md) for a template integration test.

```

Comment:
Add explicit instructions on having this repo exist in the user's `GOPATH`

Reply:
Added some notes to maintain repo in src directory.